IBPSA/Utilities/IO/SignalExchange/WeatherStation.mo

```diff
@@ -0,0 +1,316 @@
+within IBPSA.Utilities.IO.SignalExchange;
+model WeatherStation
+  "Implements typical weather measurements with signal exchange blocks"
+  IBPSA.BoundaryConditions.WeatherData.Bus weaBus "Weather data"
+    annotation (Placement(
+        transformation(extent={{-110,-10},{-90,10}}), iconTransformation(extent={{-112,
+            -14},{-86,12}})));
+  IBPSA.Utilities.IO.SignalExchange.Read reaWeaTDryBul(description=
+        "Outside drybulb temperature measurement", y(unit="K"))
+    "Outside drybulb temperature measurement"
+    annotation (Placement(transformation(extent={{-40,80},{-20,100}})));
```

Comment:
@dhblum : Can you please correct the line breaks for these instances, and make the `unit` final.

Reply:
Resolved in https://github.com/ibpsa/modelica-ibpsa/pull/1403/commits/45bb615e3c8b97590970f4619cdc2138e4442ef1 and https://github.com/ibpsa/modelica-ibpsa/pull/1403/commits/c9e43f476b51e0de0e72863ae28bafd6822bc06c.